src/audio/kpb.c

```diff
@@ -1286,9 +1280,6 @@ static enum task_state kpb_draining_task(void *arg)
 out:
 	draining_time_end = platform_timer_get(timer);
 
-	/* Reset host-sink copy mode back to unblocking */
-	comp_set_attribute(sink->sink, COMP_ATTR_COPY_TYPE, &copy_type);
```

Comment:
So instead of removing this code, please rework it so it backups the copy_type before the draining and restores it afterwards. As for the copy_mode during the draining itself - I think this copy_type should be read in during prepare method of KPB and applied on draining init.